src/Network/PacketHandlers.cs

```diff
@@ -4790,6 +4790,15 @@ private static void MegaCliloc(ref StackDataReader p)
                     continue;
                 }
 
+                int argcliloc = 0;
+
+                string[] argcheck = argument.Split('#');
```

Comment:
probably better to add a `SplitOptions.RemoveEmptyEntries` ?

Reply:
Sure, done